codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/CargoDependency.kt

```diff
@@ -200,6 +200,7 @@ data class CargoDependency(
         val HyperWithStream: CargoDependency = Hyper.withFeature("stream")
         val LazyStatic: CargoDependency = CargoDependency("lazy_static", CratesIo("1.4"))
         val Md5: CargoDependency = CargoDependency("md5", CratesIo("0.7"))
+        val Nom: CargoDependency = CargoDependency("nom", CratesIo("7"))
```

Comment:
This should go inside ServerCargoDependency.kt in the server-codegen folder as it is a dependency used only by the server right now.